lib/daru/view/adapters/googlecharts.rb

```diff
@@ -9,7 +9,7 @@ module Daru
   module View
     module Adapter
       module GooglechartsAdapter
-        extend self # rubocop:disable Style/ModuleFunction
+        extend self
```

Comment:
For ruby <2.3,  

rubocop Offense : 

`adapters/*charts.rb:11:9: C: Style/ModuleFunction: Use module_function instead of extend self.`

Rubocop offenses has to be fixed or old ruby versions. For now it can be ignored since it is just offense and not a syntax or logical error.